_posts/algorithm-development/guides/2016-10-13-git-guide.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: article
+title:  "Using Git with Algorithmia"
+excerpt: "Access Algorithmia from Git"
+date: 2016-10-13  09:50:46
+permalink: /algorithm-development/git-support/
+tags: [algo-guide-git]
+show_related: false
+author: steph_kim
+image:
+    teaser: /icons/Algorithmia_Basics.png
+---
+
+Every algorithm on Algorithmia has a dedicated git repository. This allows you to seamlessly integrate Algorithmia with your current workflows,
+such as your code editor, IDE, or deployment pipelines. 
+
+<img src="/images/post_images/legit/legit-demo.gif" alt="Demo for Git" class="screenshot">
+
+## Accessing a repository
+
+You can clone existing algorithms from the following endpoint:
+{% highlight bash %}
+    git clone https://git.algorithmia.com/git/:username/:algoname.git
```

Comment:
I'm assuming this also works with organizations ... might want to explain that here